lucille-plugins/lucille-ocr/src/main/java/com/kmwllc/lucille/ocr/stage/Rectangle.java

```diff
@@ -0,0 +1,86 @@
+package com.kmwllc.lucille.ocr.stage;
+
+import java.io.Serializable;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Rectangle implements Serializable {
```

Comment:
please fix indents and unnecessary blank lines in this file

Reply:
fixed --thanks